src/components/diamond.jsx

```diff
@@ -0,0 +1,154 @@
+import * as React from 'react';
+import styled from 'styled-components';
+
+const Angle = '30deg';
+const Width = 32;
+const Height = Width + 5;
+const ShiftPhoto = Height / 2;
+
+const EmptyDiamond = styled.div`
+    width: ${(props) => props.size || Width}rem; 
+    height: ${(props) => props.size + 5 || Height}rem; 
+`;
+
+const DiamondLeft = styled.div`
+    transform: skewY(-${(props) => props.angle || Angle});
+
+    width: ${(props) => props.size || Width}rem;
+    height: ${(props) => props.size + 5 || Height}rem;
```

Comment:
actually, it should be `height = width/sin 60`, i.e `height = width/0.866`
according to the scheme https://c1.staticflickr.com/3/2838/33642944036_43b70be683_b.jpg 
we can't leave 5 here, because this number will be different for different widths